web/docs/auth/_accessing-user-data-note.md

```diff
@@ -0,0 +1,7 @@
+<small>
+
+Read more about accessing the user data in the [Accessing User Data](/docs/auth/entities#accessing-the-auth-fields) section of the docs.
```

Comment:
> But, what I noticed then, is that this relative path will be resolved relative to where the React component is called from, and not relative to where React component is defined! Give it a try to confirm this is an issue, but from what I saw, it indeed is.

Yes, that's indeed an issue so I had to go for absolute links to be able to use snippets like this.

Reply:
Right, but if I am correct, absolute links will not work across different doc versions! THey will work only for the "current" docs, but once you give them version number and they become a bit older, they won't work. That is what this link I provided speaks about.
So at the moment when these docs end up being at `/docs/v0.13` instead of just `/docs`, that link will stop working (I think). And that means absolute URLs also don't work.
Which is tricky, I don't know the solution to this.

Reply:
I think I found a solution for this problem: "root relative links" 

Instead of writing it like this: `/docs/auth/entities`, I've written it like this `/auth/entities/entities.md`.

It's still an absolute link, but it's absolute to the "files root", not the "website" and includes the extension.

It works for me:
1. Visit:          `/docs/0.13.5/auth/username-and-pass#using-auth` and click on the link
2. It leads to `/docs/0.13.5/auth/entities#accessing-the-auth-fields`
3. Visit           `/docs/auth/username-and-pass` and click on the link
4. It leads to `/docs/auth/entities#accessing-the-auth-fields`

The `0.13.5` version is something I created locally to test out this link. I found this in the Hasura writing guide 😮‍💨  https://hasura.io/docs/3.0/wiki/docusaurus-mdx-guide/links/

Reply:
Ok perfect! Nice job, Resolve this issue if you implemented it.